### DISCUSSION TITLE
entities unavailable after upgrade to 3.6.3

### DISCUSSION THREAD

**Discussion by derSchreifritz** [emsesp/EMS-ESP32#1447] [Q&A] [answered]
hi
I did the upgrade from 3.6.2 to 3.6.3, and a few entities became unavailable, for example thermostat_hc1.
I have no idea about mqtt, but tried to follow the message from breaking changes. Downloaded mqtt explorer, connected and clicked on some trash icons. Didn't help. The editable datetime seemed to be there though. Any suggestions?

**Comment by proddy** [comment, marked as answer]
There was a bug that we just patched. Download 3.6.3 again and please try it out. https://github.com/emsesp/EMS-ESP32/releases/tag/v3.6.3

**Reply by derSchreifritz** [reply]
thanks, I think this solved it.

**Reply by derSchreifritz** [reply]
actually, I still have an issue, I can see the thermostat now, but it is not behaving as I was used to. Adjusting in homeassistant the entity climate.thermostat_hc1 does not do anything anymore, the value now called target stays. It changes only when I change number.thermostat_hc1_manualtemp
When I dial the physical box in the room, it also does change.
Is there a way to get the behavior back for the virtual thermostat?

edit: this sounds related to https://github.com/emsesp/EMS-ESP32/issues/1450

**Reply by proddy** [reply]
yes, looks like #1450. Could you also comment on that and try my suggestion.
